mypy.ini

```diff
@@ -1,7 +1,7 @@
 [mypy]
 
 strict = True
-disable_error_code = union-attr,return
```

Comment:
I spotted this while looking into https://github.com/simplistix/sybil/issues/140#issuecomment-2399383340.

I think it would be valuable to get rid of this line overall, but I don't have the understanding to get rid of `union-attr`.